resources/AI_and_data_science/DL/README.md

```diff
@@ -0,0 +1,7 @@
+# Glosarium
+Berfungsi agar bila kita ingin mencari suatu topik khusus, gak perlu buka semua resourcenya satu per satu. Berisi deskripsi singkat mengenai resource yang ada di folder ini. Bila ingin menkontribusi bacaan jangan lupa di buat ringkasannya dan di tambahkan di sini.
+
+1) How to make the most out of BERT fine tuning
+```
+BERT adalah tokenizer/language processor yang cukup populer dikalangan machine learning engineer. Selain mempermudah hidup dengan gak buat tokenizer sendiri, BERT juga di kostumisasi sesuai dengan bahasa yang kita hadapi. Namun seringkali walaupun udah pake BERT, hasilnya masih gak se impressive yang kita inginkan. Sementara kalau mau fine tuning sendiri, BERT ini agak sesnsitif, salah sedikit malah makin jelek. That's why this PDF is exist. Fine tuning yang di purpose adalah dengan memberikan early stop dan/atau mengatur setiap epoch harus diapain
+```
```

Comment:
Description like this will be better not to use a code-wrapper "\`\`\`" because it would then becomes a one line full of text and people would have to scroll through a long one liner stuff ....

![DeepinScreenshot_select-area_20210805092712](https://user-images.githubusercontent.com/18365721/128281334-d1d5ad23-9398-47ab-946f-a5fa75add70b.png)